.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+yarn pretty-quick --staged
```

Comment:
any advantage of removing it from package.json?

Reply:
The newer version requires it this way.